plugins/modules/postgresql_publication.py

```diff
@@ -775,6 +875,8 @@ def main():
 
     if tables_in_schema is not None and pg_srv_ver < 150000:
         module.fail_json(msg="Publication of tables in schema is supported by PostgreSQL 15 or greater")
+    if columns is not None and pg_srv_ver < 150000:
+        module.fail_json(msg="Publication of columns is supported by PostgreSQL 15 or greater")
```

Comment:
```suggestion
    if columns and pg_srv_ver < 150000:
        module.fail_json(msg="Publication of columns is supported by PostgreSQL 15 or greater")
```
Does it makes sense to pass an empty dict to the argument or we can simplify this a bit like suggested?